pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -282,7 +283,7 @@ func (ca *cloudCIDRAllocator) updateCIDRAllocation(nodeName string) error {
 	}
 	if needUpdate {
 		if node.Spec.PodCIDR != "" {
-			klog.ErrorS(nil, "PodCIDR being reassigned!", "nodeName", node.Name, "node.Spec.PodCIDRs", node.Spec.PodCIDRs, "cidrStrings", cidrStrings)
+			klog.ErrorS(nil, "PodCIDR being reassigned", "nodeName", node.Name, "node.Spec.PodCIDRs", node.Spec.PodCIDRs, "cidrStrings", cidrStrings)
```

Comment:
we should be consistent

"node.Name", node.Name

Reply:
Let us keep `nodeName`